### ISSUE TITLE
Migration logs some errors during accept

### ISSUE THREAD

**Issue by areebrtc** [WtfJoke/rtc2git#156]
My migration takes about 3-4 days to run, and it is still running. However, I am seeing some errors regarding 'accept'. The following form of logging is reported as below: -

To /data/STREAM/myGitRepo.git
   a1c4641..04045ad  HEAD -> CC-6.1.0.2-Stream_branchpoint
Problem running 'accept':

Successfully loaded items into the sandbox.
# On branch _branchpoint
nothing to commit, working directory clean

20:17:14 - No changes for commiting in git detected, going to reload the workspace

It is something which could hinder migration process and result in less accurate migration having missing file and directories after migration is completed. 

Please comment on this.

**Comment by WtfJoke** [comment]
Do you have more logs of Problem running 'accept'? 
Usually its written there what happen, unless the workspace contains too much conflicts

**Comment by areebrtc** [comment]
Yes I saw log that only mentions Problem running 'accept' and nothing else is there which point to some reason of failure and it is happening frequently. Also, I saw that there were conflicts in 4 files when I stopped migration and manually ran 
"scm show conflicts"
 
Can you suggest how to proceed from here? Do we need to resolve conflicts manually every-time it happens or enabling auto-conflict resolution can solve the purpose?
